package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@rubixibuc/rerender",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
We don't have to increment the version manually, the npm publishing process bumps it :)